ios/LayoutReanimation/REASharedTransitionManager.h

```diff
@@ -0,0 +1,15 @@
+#import <RNReanimated/REAAnimationsManager.h>
+#import <RNReanimated/REASnapshot.h>
+
+@interface REASharedTransitionManager : NSObject
+
+- (void)notifyAboutNewView:(UIView *)view;
+- (void)viewsDidLayout;
+- (BOOL)setupSyncSharedTransitionForViews:(NSArray<UIView *> *)views;
```

Comment:
Can we rename this to `startSharedTransitionForViews`, like on Android?

Reply:
I think that in this case, this name is better because on Android we always start Shared Transition synchronously, but on iOS, we can start it immediately or with delay (we need to wait on layout computation) so I think that different name shows well the difference between Android and iOS implementation.

Reply:
But maybe other name will be better.